deliver/lib/deliver/options.rb

```diff
@@ -189,16 +189,18 @@ def self.available_options
                                      optional: true,
                                      short_option: "-l",
                                      verify_block: proc do |value|
+                                       supported_picture_endings = [".png", ".jpg", ".jpeg"]
```

Comment:
It could probably just be a variable inside the method, just like here https://github.com/fastlane/fastlane/pull/8418/files#diff-cec46a69540d57cbb9c8631d43a7a89bR7